### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by monsij**
Hope that will suffice!

Cheers,
Monsij

### CODE REVIEW

**Comment by Demfier** [comment]
@monsij Please go through [CONTRIBUTING.md](https://github.com/demfier/gitpub/tree/master/CONTRIBUTING.md) before making any PR.
You haven't created any branch for your PR and your commit changes don't really tell what did you do in those commits. So please go through contribution guidelines first.
In any case DONOT push directly to master branch

I will fix the failing tests in the mean time.

**Comment by monsij** [comment]
Sure

**Comment by Demfier** [comment]
@monsij I have reviewed you PR, please have a look at those.

Also, you can squash all your 8 commits into 1 and give it a more useful commit message. Follow these steps to do so:
1) `git reset --soft HEAD~8`
2) `git add "your files"`
3) `git commit -am "A useful commit message"`
4) `git pull upstream master`
5) `git push origin your_branch_name`.

Make a separate branch before opening a new PR.

Let me know if you face any more issues.

**Comment by monsij** [comment]
I am getting this error when i am trying to push to my branch
_remote: Permission to demfier/gitpub.git denied to monsij.
fatal: unable to access 'https://github.com/demfier/gitpub.git/': The requested URL returned error: 403_
Please help

**Comment by Demfier** [comment]
@monsij -  I already told you. DONOT push into master directly, make a seperate branch and push your changes there. I guess you haven't cloned the repo yet or something like that. Please follow the steps in CONTRIBUTING.md, I can assure you of that you will not run into any errors while making a PR.

Cheers!

File: README.md [hunk 1]
```diff
@@ -1,30 +1,39 @@
...
+# GitPub - Python library to interact with github's public API
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
 
-Requirements:
+
+## Getting started!
+Follow up these instructions if you want to see it in action up and running on your machine.
+
+### <em>System Requirements</em>:
 <br>------------------
```
[the remark this answers is not in the document]
Comment (monsij) [reply README.md]: i'm skipping it

**Pull request closed without merging**